packages/xod-arduino/src/transpiler.js

```diff
@@ -540,15 +540,17 @@ const checkForNativePatchesWithTooManyOutputs = def(
   }
 );
 
-// :: PatchPath -> Project -> [BoundLiteral]
-const listBoundLiterals = R.curry((curPatchPath, project) =>
+const getGlobalLiteralsFromTProject = def(
+  'getGlobalLiteralsFromTProject :: TProject -> [String]',
   R.compose(
+    R.map(R.tail), // get rid of leading `=`
```

Comment:
You get rid of `=`. So, they are **globals**, not literals. `getGlobalLiteralsFromTProject` → `getGlobalNamesFromTProject`